src/main/java/com/aspire/student/controller/StudentController.java

```diff
@@ -211,7 +211,8 @@ public void getAllStudents(ResourceRequest request, ResourceResponse response) {
       logger.info("Getting all student list");
       studentList = studentService.getStudentList();
     } catch (Exception e) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://13.59.219.140:9000/coding_rules#rule_key=squid%3AS1166)